test/end-to-end/pages/CreateCase.js

```diff
@@ -154,8 +163,15 @@ module.exports = {
     await this.clickCreateCase();
     await this.fillFormAndSubmit_TS();
     await this.selectTypeOfAdminNocApplicationC100();
-    await I.retry(retryCount).click('Create my dummy case');
-    // await I.wait('10');
+    await I.click('Create my dummy case');
+    await this.amOnHistoryPageWithSuccessNotification();
+  },
+
+  async createFLCaseByCourtAdmin() {
```

Comment:
instead to refer FL please use FL401 or DA for these type of case

Reply:
@balaganesanhmcts you ok with me changing the name in my next PR which I am planning to push this noon ?

Reply:
Ok